src/components/Connections/connectionTypes.tsx

```diff
@@ -1807,6 +1807,69 @@ export const connectionTypes: ConnectionType[] = [
         }
       };
     }
+  },
+  {
+    title: "Git",
+    icon: "git",
+    value: ConnectionValueType.Git,
+    fields: [
+      {
+        label: "Name",
+        key: "name",
+        type: fieldTypes.input,
+        required: true
+      },
+      {
+        label: "URL",
+        key: "url",
+        type: fieldTypes.input,
+        required: true
+      },
+      {
+        label: "Username",
```

Comment:
Its either Username or Certificate, not both and I don't think any of them are required.